### PR TITLE
Hotfix: use pure javascript crypto lib

### PR DESCRIPTION
**Pull request by itxtoledo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
   "name": "biscoint-api-node",
-  "version": "1.2.1",
+  "version": "1.2.2",
```
Comment (bolaum) [inline package.json:3]: versions in `package-lock.json` and `package.json` are different

**Review by bolaum — CHANGES_REQUESTED** [review]
Comment inline

File: package.json (hunk 1 above)
Comment (itxtoledo) [reply package.json:3]: ok

Comment (itxtoledo) [reply package.json:3]: solved

**Pull request merged**